runtime/common/tests/xcm_mock/parachain.rs

```diff
@@ -18,7 +18,7 @@
 
 use codec::{Decode, Encode};
 use frame_support::{
-    assert_ok, construct_runtime,
+    assert_ok, construct_runtime, match_type,
```

Comment:
This macro is going to be deprecated, please `match_types`.

Reply:
This branch is still on 0.9.18